vitrina/users/views.py

```diff
@@ -28,7 +29,19 @@ class RegisterView(CreateView):
     def post(self, request, *args, **kwargs):
         form = self.form_class(request.POST)
         if form.is_valid():
-            user = form.save()
+            user = form.save(commit=False)
+            if Representative.objects.filter(email=user.email).exists():
```

Comment:
There are multiple ways, how this can go wrong, same user can be added to multiple organizations, and you adding representative to a random first one. User might change his email leaving it open for grabs for someone else...

Let's not rely on `email`, instead use secure one time token with an expiration date. You can use [It's Dangerous](https://itsdangerous.palletsprojects.com/) for this.

Also it would be nice to decouple this from users, since this is orgs feature. You can do the decoupling using [signals](https://docs.djangoproject.com/en/3.2/ref/signals/).